images/base/Dockerfile.centos7

```diff
@@ -4,9 +4,9 @@
 #
```

Comment:
Ok i believe the only difference is the rpm. @tdawson you ok with merging these two.